cmd/notify-mailer/main.go

```diff
@@ -130,68 +140,68 @@ func (m *mailer) run() error {
 
 	startTime := m.clk.Now()
 
-	for i, dest := range destinations {
-		m.printStatus(dest, i+1, len(destinations), startTime)
-		if strings.TrimSpace(dest) == "" {
+	sortedAddresses := sortAddresses(addressesToRecipients)
+
+	var sent int
+	for i, address := range sortedAddresses {
+		if !m.targetRange.includes(address) {
+			m.log.Debugf("skipping %q: out of target range")
 			continue
 		}
-		err := m.mailer.SendMail([]string{dest}, m.subject, m.emailTemplate)
+		recipients := addressesToRecipients[address]
+		i++
```

Comment:
There is nothing technically wrong with this but incrementing the range index value feels... wrong. Any reason not to just use `i+1` in the print statement?

Reply:
Actually, this was technically wrong - in one iteration I didn't have an `i` in the range statement so added this. Thanks for catching!